docs/src/main/paradox/common/binary-compatibility-rules.md

```diff
@@ -60,15 +54,17 @@ Some modules are excluded from the binary compatibility guarantees, such as:
  
 ### When will a deprecated method be removed entirely
 
-Once a method has been deprecated then the guideline* is that it will be kept, at minimum, for one **full** minor version release. For example, if it is deprecated in version 2.5.2 then it will remain through the rest of 2.5, as well as the entirety of 2.6. 
+Once a method has been deprecated then the guideline* is that it will be kept, at minimum, for one **full** minor version release. For example, if it is deprecated in version 1.0.2 then it will remain through the rest of 1.0, as well as the entirety of 1.1.
+
+Methods that were deprecated in Akka, before the project fork to Pekko, are being considered for removal in Pekko 1.1.0.
```

Comment:
Sorry I meant removed instead of reviewed (i.e. I was under impression that we would remove all deprecated methods in 1.1.x).

I don't think there is a formal decision, but we can make a `[VOTE]` thread on the mailing list

Reply:
> Since the current text in this PR is non-commital, could we consider it as a useful guideline.

Fine with me, consider this resolved